utils.go

```diff
@@ -0,0 +1,17 @@
+package thousandeyes
+
+// Bool is a helper routine that allocates a new bool value
+// to store v and returns a pointer to it.
+func Bool(v bool) *bool { return &v }
```

Comment:
Is this a common practice in Go?

Reply:
I'm not sure if it's common, but I think it makes sense for our use-case. I took it verbatim from [Google's go-github API wrapper library.](https://github.com/google/go-github/blob/7a7a7f0/github/github.go#L1352-L1366)